parsec

```diff
@@ -17,8 +17,15 @@ echo "launching parsec: (u${USERID}:g${GROUPID})"
 docker run -it --rm --init \
   --env DISPLAY \
   --env PULSE_SERVER \
+  --privileged \
```

Comment:
Do we know for sure if we need this? I believe passing over the USB devices should suffice